src/WebJobs.Script/Binding/EasyTableBinding.cs

```diff
@@ -29,6 +32,10 @@ public class EasyTableBinding : FunctionBinding
 
         public string Id { get; private set; }
 
+        public string MobileAppUri { get; private set; }
+
+        public string ApiKey { get; private set; }
+
```

Comment:
@mathewc -- is there a reason that these all of these bindings need to have public properties like this? I was tempted to remove them and just store the metadata as a private field. But I wasn't sure if something somewhere consumed them...